src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+
+  &__product-image {
+    padding: 16px 3px 0;
```

Comment:
Padding property should be set in a clockwise direction (top, right, bottom, left). The correct value should be `padding: 16px 3px 0 3px;`